packages/valory/skills/decision_maker_abci/behaviours/storage_manager.py

```diff
@@ -276,9 +276,17 @@ def _update_accuracy_store(self, local_tools: List[str]) -> None:
         )
         accuracy_store = self.policy.accuracy_store
 
+        # remove tools which are irrelevant
+        for tool in accuracy_store.copy():
+            if tool not in self.mech_tools:
+                accuracy_store.pop(tool, None)
+
         # update the accuracy store using the latest accuracy information (only entered during the first period)
         for row in reader:
             tool = row[self.acc_info_fields.tool]
+            if tool not in accuracy_store.keys():
+                continue
```

Comment:
what is the rationale behind this? Ignoring a tool not present at the local store?

Reply:
Exactly. Keep in mind the store only contains the relevant tools.